status/types/chat.nim

```diff
@@ -14,11 +14,9 @@ type ChatType* {.pure.}= enum
   Public = 2,
   PrivateGroupChat = 3,
   Profile = 4,
-  Timeline = 5
-  CommunityChat = 6
+  CommunityChat = 5
```

Comment:
Just as mentioned in https://github.com/status-im/status-desktop/pull/4198#discussion_r758125727, we can't just treat the value `5` as the `CommunityChat` variant. This has to stay `6`

Reply:
done